typing/env.ml

```diff
@@ -389,27 +399,29 @@ module IdTbl =
 
 
     let rec find_all name tbl =
-      List.map (fun (id, desc) -> Pident id, desc)
+      List.map
+        (fun (id, (desc, _)) -> Pident id, desc)
         (Ident.find_all name tbl.current) @
       match tbl.opened with
       | None -> []
       | Some {root; using = _; next; components} ->
           try
-            let (desc, pos) = NameMap.find name components in
-            (Pdot (root, name, pos), desc) :: find_all name next
+            let (desc, _) = NameMap.find name components in
+            (Pdot (root, name), desc) :: find_all name next
           with Not_found ->
             find_all name next
```

Comment:
Like for `TycompTbl.diff_keys` the changes to this function are weird, and inconsistent with the other changes in this module: this function is making the assumption that the `'a` in `current : Ident.tbl (* in [t] *)` and in `components: 'a list NameMap.t (* in [opened] *)` is a tuple:
```ocaml
val find_all: string -> ('a * 'b) t -> (Path.t * 'a) list
```
But there's no reason to. And, for example, I believe you wouldn't be able to call `find_all` on the `modtypes` field of the environment with this implementation.
It's the callers that should change.